wp-cli/vip-two-factor.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+class VIP_Two_Factor_Command extends WPCOM_VIP_CLI_Command {
+	function report() {
+		$fields = array( 'ID', 'display_name' );
+		$users = get_users( array( 'fields' => $fields ) );
+
+		$cap = apply_filters( 'wpcom_vip_two_factor_enforcement_cap', 'edit_posts' );
+		foreach ( $users as $user ) {
+			if ( Two_Factor_Core::is_user_using_two_factor( $user->ID ) ) {
+				$user->two_factor_enabled = 'true';
+			} else if ( ! user_can( $user->ID, $cap ) ) {
+				$user->two_factor_enabled = 'n/a';
+			} else {
+				$user->two_factor_enabled = 'false';
+			}
+		}
```

Comment:
Thoughts, on including which providers are in use? (can use `Two_Factor_Core ::get_enabled_providers_for_user( $user )` to do that)

Reply:
Ah yes, good call.